### PR TITLE
testing/rclone: Update to 1.50.1

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
It fails on arm. Any advice how to fix this?

File: testing/rclone/APKBUILD [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# Contributor: Sandro Jäckel <sandro.jaeckel@posteo.de>
```
Comment (Cogitri) [inline testing/rclone/APKBUILD]: Usually contributor fields are added for more significant changes :sweat_smile:

Comment (SuperSandro2000) [reply testing/rclone/APKBUILD]: ``Usually``
I also want to bump it to 1.50.2

**Comment by maxice8** [comment]
d78143dd4728285b06d2b969fac8415d02d1e1c4

**Pull request closed without merging**